app/pages/preview/createPreviewPageContext.js

```diff
@@ -1,7 +1,16 @@
 export const createPreviewPageContext = ({ previewData }) => {
-  const context = {
+  let context = {
     sections: previewData.sections,
   };
 
+  for(let key in context.sections)
+  { 
+    let section = context.sections[key];
+    if(section.answers && section.answers['document-name'])
+    {
```

Comment:
surprised eslint didn't complain. I know this is a .NET convention but we have start on the brace on the same line

Reply:
I think it has failed in the pipeline